src/algorithms.jl

```diff
@@ -5995,9 +6004,18 @@ Assyr Abdulle. Fourth Order Chebyshev Methods With Recurrence Relation. 2002 Soc
 Industrial and Applied Mathematics Journal on Scientific Computing, 23(6), pp 2041-2054, 2001.
```

Comment:
I can't comment on the line directly, but it may be nice to add the signature with default arguments to the top of the docstring, e.g.,
```
"""
    ROCK4(; min_stages = 0, max_stages = 152, eigen_est = nothing)

Assyr Abdulle. Fourth Order Chebyshev Methods With Recurrence Relation. 2002 Society for
Industrial and Applied Mathematics Journal on Scientific Computing, 23(6), pp 2041-2054, 2001.
```